src/FormParserFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Parser;
+
+class FormParserFactory
```

Comment:
`BodyParser` sounds good to me, and then rename this class to `Factory` as it creates a parser for the determined content type.